scripts/extensions/print_keys

```diff
@@ -1,3 +1,11 @@
 #!/bin/sh
 
-erl -pa {{lib_dirs}}/**/ebin -noshell -eval 'miner_keys:print_keys("{{sys_config}}")' -eval 'init:stop()'
+CONFIG=$(find $ROOTDIR/releases -name sys.config)
+LIB_DIR=$ROOTDIR/lib
+
+if relx_nodetool "ping" > /dev/null 2>&1; then
+    relx_nodetool "rpcterms" miner_keys print_keys "'${CONFIG}'." | sed '$d'
```

Comment:
What does `sed '$d'` do?

Reply:
I don't think you want to call this same function in both contexts, if the miner is running there's already a process managing the ecc508, this call will boot a new one.

Reply:
It strips out the trailing `ok` line from the rpc result.

Reply:
> I don't think you want to call this same function in both contexts, if the miner is running there's already a process managing the ecc508, this call will boot a new one.

You are correct.  From the comments for the `keys` fucntion:

```
%% NOTE: Do NOT call this after miner has started since this function
%% will attempt to communicate directly with the ECC. Use only as part
%% of startup or other miner-free scripts.
```

So unless miner has the equivalent of `dump_keys` defined somewhere I should just revert back to `cat /var/data/public_keys` when miner is running?

Reply:
Reverted back to `cat /var/data/public_keys`.